src/GCRealTimeMon/CallStackResolution/CallStackManager.cs

```diff
@@ -50,12 +52,13 @@ public static class CallStackManager
                 ClrTraceEventParser.Keywords.JittedMethodILToNativeMap |
                 ClrTraceEventParser.Keywords.JITSymbols                |
                 ClrTraceEventParser.Keywords.JitTracing                |
+                ClrTraceEventParser.Keywords.GCAllObjectAllocation     |
```

Comment:
Worth calling out that turning on `GCAllObjectAllocation` will generate one event per allocation on runtimes that support it.  This is likely to be prohibitively expensive.  You will probably want to make this configurable.